test/feeds.test.js

```diff
@@ -0,0 +1,67 @@
+const request = require('supertest');
+
+const app = require('../src/backend/web/app');
+const Feed = require('../src/backend/data/feed');
+const { addFeed } = require('../src/backend/utils/storage');
+const hash = require('../src/backend/data/hash');
+
+describe('test /feeds endpoint', () => {
+  const createdItems = 150;
+
+  // Array of feeds
+  const feeds = [...Array(createdItems).keys()].map(item => {
+    const url = `http://telescope${item}.cdot.systems`;
+    return {
+      id: hash(url),
+      author: 'foo',
+      url,
+    };
+  });
+
+  beforeAll(() => Promise.all(feeds.map(feed => addFeed(feed))));
+
+  it('requests feeds', async () => {
+    const res = await request(app).get('/feeds');
+
+    expect(res.status).toEqual(200);
+    expect(res.get('Content-type')).toContain('application/json');
+    expect(res.body.length).toBe(createdItems);
+    expect(res.body instanceof Array).toBe(true);
+  });
+});
+
+describe('test /posts/:guid responses', () => {
+  const existingGuid = 'http://existing-guid';
```

Comment:
We don't work with `guid`s in the Feeds, so let's rename these.  Maybe `existingUrl` and `missingUrl`?